airbyte-lib/airbyte_lib/factories.py

```diff
@@ -16,9 +16,16 @@ def get_in_memory_cache():
     return InMemoryCache()
 
 
-def get_connector(name: str, version: str = "latest", config: Optional[Dict[str, Any]] = None):
+def get_connector(name: str, version: str = "latest", config: Optional[Dict[str, Any]] = None, use_venv: bool = True):
+    """
+    Get a connector by name and version.
+    :param name: connector name
+    :param version: connector version - if not provided, the most recent version will be used
+    :param config: connector config - if not provided, you need to set it later via the set_config method
+    :param use_venv: whether to use a virtual environment to run the connector. If False, the connector is expected to be available on the path (e.g. installed via pip). If True, the connector will be installed in a virtual environment.
+    """
```

Comment:
I read "use_venv" the first time to mean that "false" would still be managed, but would be installed by AirbyteLib without venv isolation. Wdyt of a name like "already_installed" or "needs_install" or similar to distinguish AirbyteLib-managed installations vs user-managed?

Reply:
what about "auto_install"?

Reply:
I merged already, I will implement this on the base PR